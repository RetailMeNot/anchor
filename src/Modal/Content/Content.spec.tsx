// REACT
import * as React from 'react';
import * as renderer from 'react-test-renderer';

// STYLED COMPONENTS
/* tslint:disable no-import-side-effect*/
import 'jest-styled-components';
/* tslint:enable */

// COMPONENT
import { Modal } from '..';
const { Content } = Modal;

describe('Component: Modal.Content', () => {
    it('should render', () => {
        const subject = <Content>Text</Content>;
        const tree = renderer.create(subject).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
