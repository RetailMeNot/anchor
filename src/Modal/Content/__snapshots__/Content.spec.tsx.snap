// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Modal.Content should render 1`] = `
.c0 {
  box-sizing: border-box;
  width: 100%;
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<div
  className="anchor-modal-content c0"
>
  Text
</div>
`;
